docs/source/components/nodes/color_camera.rst

```diff
@@ -46,6 +56,11 @@ Inputs and Outputs
 into :ref:`NeuralNetwork`. :code:`video` and :code:`still` are both NV12, so are suitable for bigger sizes. :code:`still` image gets created when
 a capture event is sent to the ColorCamera, so it's like taking a photo.
 
+**ISP** (image signal processor) is used for bayer transformation, demosaicing, noise reduction etc. Click
+`here <https://en.wikipedia.org/wiki/Image_processor>`__ for more information.
```

Comment:
Could add that ISP interacts with the 3A algorithms (auto-focus, auto-exposure, auto-white-balance), adjusting at runtime the image sensor exposure time and sensitivity (ISO), and the lens position if motorized.